internal/accountstable/accountstable.go

```diff
@@ -14,6 +15,86 @@ import (
 	"github.com/SIGBlockchain/project_aurum/internal/sqlstatements"
 )
 
+// structs
+
+// ATConnection struct used for
+type ATConnection struct {
+	lock   sync.RWMutex
+	dbconn *sql.DB
+}
+
+// Lock locks at for writing. If the lock is already locked for reading or writing,
+// Lock blocks until the lock is available.
+func (at *ATConnection) Lock() {
+	at.lock.Lock()
+}
+
+// Unlock unlocks at for writing. It is a run-time error if at is not locked for
+// writing on entry to Unlock.
+func (at *ATConnection) Unlock() {
+	at.lock.Unlock()
+}
+
+// InsertAccountIntoAccountBalanceTable is called from receiver in order to enforce interface
```

Comment:
"Calls accountstable.InsertAccountIntoAccountBalanaceTable using struct connection"

Same for all other TODO's; just replace the function name